### PR TITLE
Update try-or-buy-microsoft-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the navigation changes to buy products as it is no longer purchase services as it is now in the home page -->marketplace

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9292

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/try-or-buy-microsoft-365.md [hunk 1]
```diff
@@ -95,8 +95,8 @@ If you already have a Microsoft 365 for business subscription, you can use the M
 
 When you add a subscription through the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">Microsoft 365 admin center</a>], the new subscription is associated with the same organization (domain namespace) as your existing subscription. This association makes it easier to move users in your organization between subscriptions, or to assign them a license for the additional products they need.
 
-1. In the admin center, go to the **Billing** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=868433" target="_blank">Purchase services</a> page.
-2. On the **Purchase services** page, you see the plans that are available to your organization. Choose the Microsoft 365 plan that you want to try.
+1. In the admin center, go to the **Marketplace**
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/try-or-buy-microsoft-365.md]:
```suggestion
1. In the admin center, go to **Marketplace**.
```

File: microsoft-365/commerce/try-or-buy-microsoft-365.md [hunk 2]
```diff
@@ -95,8 +95,8 @@ If you already have a Microsoft 365 for business subscription, you can use the M
...
+2. On the **Marketplace** page, you see the plans that are available to your organization. Choose the Microsoft 365 plan that you want to try.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/try-or-buy-microsoft-365.md]:
```suggestion
2. On the **Marketplace** page, you'll see the plans that are available to your organization. Choose the Microsoft 365 plan that you want to try.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by cmcatee-MSFT** [comment]
@msbemba & @yogkumgit I've found no evidence that **Purchase services** has changed to **Marketplace** in the Microsoft 365 admin center. The article doesn't reference a "purchase subscription Tab" as the customer claimed.

**Pull request closed without merging**